### PR TITLE
Search more intra modes in inter frames

### PR DESCRIPTION
**Pull request by fbossen**
Reuse exisiting intra mode lists to determine which intra modes
to search in an inter frame

### CODE REVIEW

**Comment by fbossen** [comment]
|    PSNR | PSNR Cb | PSNR Cr | PSNR HVS |    SSIM | MS SSIM | CIEDE 2000 |
|    ---: |    ---: |    ---: |     ---: |    ---: |    ---: |       ---: |
| -2.5792 |  1.3078 |  1.5778 |  -2.4461 | -2.6847 | -2.9884 |    -1.3680 |

File: src/predict.rs [hunk 1]
```diff
@@ -38,7 +38,6 @@ pub static RAV1E_INTRA_MODES_MINIMAL: &'static [PredictionMode] = &[
 
 pub static RAV1E_INTER_MODES: &'static [PredictionMode] = &[
   PredictionMode::GLOBALMV,
-  PredictionMode::DC_PRED,
```
Comment (lu-zero) [inline src/predict.rs]: Maybe could be made a separate commit to tell why.

Comment (fbossen) [reply src/predict.rs]: In this PR the list of modes tested in inter frames is constructed from the combination of a list of inter modes and a list of intra modes. There is thus no need anymore to list intra modes in the list of inter modes.

**Comment by ycho** [comment]
Is the awcy results above from default speed, i.e. 3?
If yes, has -s 0 and 1 test clear? :)

**Comment by fbossen** [comment]
@ycho Results are for the default speed (3). I haven't tested at speeds 0 and 1 (I don't expect an issue specific to small blocks here), but can do.

**Comment by ycho** [comment]
Just tested for -s 0 and 1, and all fine. Just it now is SUPER SLOWER!! Let me land..
